src/js/grapl-cdk/bin/deployment_parameters.ts

```diff
@@ -2,10 +2,11 @@ import * as ec2 from '@aws-cdk/aws-ec2';
 
 module HardcodedDeploymentParameters {
     // ex: 'Grapl-my-deployment'
+
     export const deployName = undefined;
 
     // defaults to 'latest'
-    export const graplVersion = undefined;
+    export const graplVersion = 'latest';
```

Comment:
This should remain `undefined`. If it doesn't indeed default to latest below as the docstring indicates then we should fix that.